airflow/models.py

```diff
@@ -1494,8 +1494,13 @@ def get_template_context(self, session=None):
         prev_execution_date = task.dag.previous_schedule(self.execution_date)
         next_execution_date = task.dag.following_schedule(self.execution_date)
 
-        next_ds = next_execution_date.strftime('%Y-%m-%d')
-        prev_ds = prev_execution_date.strftime('%Y-%m-%d')
+        next_ds = None
```

Comment:
about just set it to tomorrow_ds

Reply:
Would sort of work, but could break assumption /shrug

Probably need to update the docs accordingly then.

Reply:
another way is to actually ensure that next_execution_date is always not None